extensions/lifecycle/objectProcessor/LifecycleObjectExpirationProcessor.js

```diff
@@ -33,9 +33,10 @@ class LifecycleObjectExpirationProcessor extends LifecycleObjectProcessor {
      * @param {Number} s3Config.port - s3 endpoint port
      * @param {String} [transport="http"] - transport method ("http"
      *  or "https")
+     * @param {Object} gcConfig - gc congifuration object
      */
-    constructor(zkConfig, kafkaConfig, lcConfig, s3Config, transport = 'http') {
-        super(zkConfig, kafkaConfig, lcConfig, s3Config, transport);
+    constructor(zkConfig, kafkaConfig, lcConfig, s3Config, transport = 'http', gcConfig) {
```

Comment:
gcConfig should probably be added before the `transport` field : otherwise we may have issues using the default value for that field (and it seems more consistent to keep the `config` fields together)